app/config/mq-config.js

```diff
@@ -126,6 +133,13 @@ const mqConfig = {
     username: process.env.MESSAGE_QUEUE_USER,
     password: process.env.MESSAGE_QUEUE_PASSWORD,
     type: 'sessionQueue'
+  },
+  responseEligibilityQueue: {
+    name: process.env.ELIGIBILITY_CHECK_RESPONSE_QUEUE_NAME,
+    address: process.env.ELIGIBILITY_CHECK_RESPONSE_QUEUE_ADDRESS,
+    username: process.env.MESSAGE_QUEUE_USER,
+    password: process.env.MESSAGE_QUEUE_PASSWORD,
+    type: 'sessionQueue'
```

Comment:
I don't think we actually need this `type` property as it's only for subscribing to queues and topics.  But we're not subscribing, we're accepting a session